### ISSUE TITLE
yarn complains about unmet peer dependency

### ISSUE THREAD

**Issue by danr** [fathyb/parcel-plugin-typescript#29]
Hi fathyb,

Yarn keeps giving me this warning:
```
warning " > parcel-plugin-typescript@0.7.0" has unmet peer dependency "typescript@^2.4.0".
```

Is there something I could do about this?

I'm using `"typescript": "^2.7.1"` in my package.json.

Thanks,
Dan

**Comment by fathyb** [comment]
I've opened [yarnpkg#4850](https://github.com/yarnpkg/yarn/issues/4850) a moment ago for a related issue with Yarn and peer dependencies. Could try using `npm` just to check if it's the same bug or legit problem on our side?

**Comment by danr** [comment]
I don't get anything similar using `npm`.

**Comment by fathyb** [comment]
Then I'm afraid it's the aforementioned `yarn` issue. I'm also affected on multiple projects (not even using Parcel) and did not found any workaround.
